src/quo2/components/navigation/page_nav/view.cljs

```diff
@@ -0,0 +1,285 @@
+(ns quo2.components.navigation.page-nav.view
+  (:require [quo2.components.avatars.group-avatar.view :as group-avatar]
+            [quo2.components.buttons.button.view :as button]
+            [quo2.components.dropdowns.dropdown :as dropdown]
+            [quo2.components.icon :as icons]
+            [quo2.components.markdown.text :as text]
+            [quo2.components.navigation.page-nav.style :as style]
+            [quo2.theme :as theme]
+            [react-native.core :as rn]))
+
+(def ^:private button-type
+  {:white       :grey
+   :neutral-5   :dark-grey
+   :neutral-90  :grey
+   :neutral-95  :dark-grey
+   :neutral-100 :black
+   :photo       :grey
+   :blur        :grey})
+
+(defn- page-nav-base
+  [{:keys [margin-top background on-press accessibility-label icon-name]
+    :or   {background :white}}
+   & children]
+  (into [rn/view {:style (style/container margin-top)}
```

Comment:
I have mentioned elsewhere in this pr but consider adding a generic `container-style` prop that is merged with `style/container` so we can set margins/paddings etc on this component, it aligns with the other components then 👍

Reply:
Done!